plugins/orchestrator/README.md

```diff
@@ -0,0 +1,201 @@
+- OpenAPI / REST built-in integration etc.
+
+**Client-side tooling**
+
+- Orchestration visualization / graphical editor
+- Integration with service catalog/actions
+- GitHub integration
+- Form generation
+- Runtime monitoring of instances
+- Dashboards
+- Potential custom integrations (user interaction, notifications, etc.)
+
+## For administrators
+
+### Installation
+
+The Orchestrator plugin is composed of the following packages:
+
+- `@janus-idp/backstage-plugin-orchestrator-backend` package connects the Backstage server to the Orchestrator. For setup process, see [Backend Setup](#setting-up-the-orchestrator-backend-package)
+- `@janus-idp/backstage-plugin-orchestrator` package contains frontend components for the Orchestrator plugin. For setup process, see [Frontend Setup](#setting-up-the-orchestrator-frontend-package)
+- `@janus-idp/backstage-plugin-orchestrator-common` package contains shared code between the Orchestrator plugin packages.
+- `@janus-idp/backstage-plugin-catalog-backend-module-orchestrator-entity-provider` package is a backend module to the catalog plugin to build the Orchestrator entity provider.
+
+#### Prerequisites
+
+- Docker up and running (currently it is a limitation, see [Limitations](#limitations))
+
+#### Setting up the configuration for the Orchestrator plugin
+
+The following configuration is required for the Orchestrator plugin to work properly:
+
+```yaml title="app-config.yaml"
+orchestrator:
+  sonataFlowService:
+    baseUrl: http://localhost
+    port: 8899
+    workflowsSource:
+      gitRepositoryUrl: https://github.com/tiagodolphine/backstage-orchestrator-workflows
+      localPath: /tmp/orchestrator/repository
+      autoPush: true
+```
+
+For more information about the configuration options, including other optional properties, see the [config.d.ts](../orchestrator-common/config.d.ts) file.
+
+- Although optional, you may also want to set up the `GITHUB_TOKEN` environment variable to allow the Orchestrator to access the GitHub API.
+
+#### Setting up the Orchestrator backend package
+
+1. Install the Orchestrator backend plugin using the following command:
+
+   ```console
+   yarn workspace backend add @janus-idp/backstage-plugin-orchestrator-backend
+   ```
+
+1. Create a new plugin instance in `packages/backend/src/plugins/orchestrator.ts` file:
+
+   ```ts title="packages/backend/src/plugins/orchestrator.ts"
+   import { Router } from 'express';
+
+   import { createRouter } from '@janus-idp/backstage-plugin-orchestrator-backend';
+
+   import { PluginEnvironment } from '../types';
+
+   export default async function createPlugin(
+     env: PluginEnvironment,
+   ): Promise<Router> {
+     return await createRouter({
+       eventBroker: env.eventBroker,
+       config: env.config,
+       logger: env.logger,
+       discovery: env.discovery,
+       catalogApi: env.catalogApi,
+       urlReader: env.reader,
+     });
+    }
+   }
```

Comment:
seems like this curly brace is redundant

Reply:
Thanks @rgolangh! Fixed in https://github.com/janus-idp/backstage-plugins/pull/783/commits/2826f33cf9fb161d2185f05b7952adfc4bef1ef1.